assets/quick-order-list.js

```diff
@@ -160,23 +161,53 @@ class QuickOrderList extends HTMLElement {
     ];
   }
 
-  renderSections(parsedState) {
+  renderSections(parsedState, id) {
```

Comment:
Instead of passing in the `id` here, I wonder if we should replace this here instead:

https://github.com/Shopify/dawn/blob/main/assets/quick-order-list.js#L136-L138

with the row you want updated

Reply:
Thanks for idea Sofia. I gave it a try and I think it's not worth to do it. The main reason is we still need to re-render the whole section when we Remove All. So, I'd leave it as is now.